core_torchbench_ops.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD 3-Clause license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
```

Comment:
Can you create a constants file for this stuff

Reply:
Also I think there's a way to get these without hardcoding them. Ask @msaroufim for a pointer / look through the scripts folder.